keras_cv/layers/preprocessing3d/global_random_flip_y.py

```diff
@@ -50,14 +53,20 @@ def augment_point_clouds_bounding_boxes(
 
         point_clouds = tf.concat([point_clouds_x, point_clouds[..., 1:]], axis=-1)
 
-        bounding_boxes_x = -bounding_boxes[..., 0:1]
-        bounding_boxes_heading = wrap_angle_radians(-bounding_boxes[..., 6:7])
+        bounding_boxes_x = -tf.expand_dims(
```

Comment:
What's the benefit of using expand_dims VS bounding_boxes[..., CENTER_XYZ_DXDYDZ_PHI.X:CENTER_XYZ_DXDYDZ_PHI.X+1] ?

either way is fine

Reply:
Sounds good. Feel free to use expand_dims. No need to change it